plugin-test/src/test/groovy/org/gradle/github/dependency/extractor/BaseExtractorTest.groovy

```diff
@@ -183,7 +190,9 @@ abstract class BaseExtractorTest extends Specification {
                 def actual = resolved[key]
                 def expected = expectedResolved[key]
 
-                assert actual.package_url == expected.package_url
+                if (expected.package_url != null) {
+                    assert actual.package_url == expected.package_url
+                }
```

Comment:
Might you want to assert that it isn't present, rather than not asserting anything at all?

Reply:
No, the package_url will be present, but this test isn't asserting anything about it.

Reply:
Might be worth continuing to assert that it is set to a non-empty value, beginning with `pkg:`, even if you're not asserting anything else about it?